_posts/2019-07-15-hccamp.md

```diff
@@ -0,0 +1,82 @@
+
+## How it all began
+
+With the goal of building one strong IT, two years ago our whole IT met during a couple of days to discuss and share views on different topics around HolidayCheck, IT, design, the travel industry and much more. This get-together was knwon as AllDevs and was the beginning of a great tradition.
+
+This has evolved during last two editions involving more departments related to the IT organisation (Product, UX, Agile), different locations, colleagues from other companies (this last edition friends joined us from DriveBoo, Weeronline and Workpath) and new conference styles. And that brings us to what we are today, the HolidayCheckCamp or HC.Camp.
+
+But why calling it a Camp?
+
+- the camp is a place to meet, share and discuss.
+- it introduces the nomadic view of the conference; each year, one different location.
+- the naming does not limit the participants to being only devs.
+
+## HolidayCheckCamp 2019
+
+This last edition has taken place in Bulgaria, at the Golden Sands, in the HolidayCheck Award 2019 [Hotel Griffid Encanto Beach](https://www.holidaycheck.de/hi/grifid-encanto-beach/22201c5d-e735-3966-803a-b44572669ffa). Due to the previous work of an amazing Orga Team, the HC.Camp was an awesome experience for me, both as organiser and participant. 
+
+(missing some text here)
+
+The conference itself consisted of 2 different tracks: Campfire and Compass.
+
+### Campfire Track
+
+Following an open space format, both conference days started with a welcoming speech and a Marketplace. Every camper introduced to the others one or more topics that he/she was interested about and placed it in the schedule wall, including 5 different spaces and 7 time slots per day.
+
+There are some basic rules to keep in mind for an open space:
+
+- Whoever comes is the right people
+- Whatever happens is the only thing that could’ve
+- When it starts is the right time
+- When it’s over, it’s over 
+
+We had more than 60 different topics to discuss about, so exciting as:
+
+- how to make our product **faster** and more **performant**
+- how to bring **sustainability** to the travel industry
+- how to improve the **transparency** between management and teams
+- how could we improve the **collaboration** between UX and developers
+- what is the future of **mobile and cross-device development** at HolidayCheck
+- how to do **urlauber-friendly ads**
+- we did some **hacking** and **coding** sessions
+- and much more: **testing, innovation, deep learning, data and services, visual development or meeting culture**
+- and we even had the chance to learn some **salsa** and **charleston** moves or get to know some insides about the **Indian culture** from our Indian colleagues
+
+### Compass track
+
+New this year we did a *classic* conference track where our great HolidayCheck people could bring out the speaker in them and present us the topic they are passionate about. 
```

Comment:
You could write that we tried to experiment with previous formula a bit because this is why we mixed compass and campfire probably :)

Reply:
I tried to formulate differently 👍